src/plugins/vis_type_timeline/server/series_functions/graphite.test.js

```diff
@@ -90,7 +90,7 @@ describe('graphite', function () {
     });
   });
 
-  it('should return error message if both allowlist and blocklist are disabled', function () {
```

Comment:
we can probably keep this the same and then add another set of tests that utilize the deny property.